.github/workflows/spark_sql_test_ansi.yml

```diff
@@ -22,17 +22,17 @@ concurrency:
   cancel-in-progress: true
 
 on:
-  # enable the following once Ansi support is completed
-  #  push:
-  #    paths-ignore:
-  #      - "doc/**"
-  #      - "**.md"
-  #  pull_request:
-  #    paths-ignore:
-  #      - "doc/**"
-  #      - "**.md"
-
-  # manual trigger ONLY
+  push:
+    paths-ignore:
+      - "doc/**"
+      - "docs/**"
```

Comment:
we need to have only 1 docs folder..... its not related to this PR thoug

Reply:
removed doc